### PR TITLE
fix Hokuyo namespace to be consistent with the SICK lasers

### PR DESCRIPTION
**Pull request by frederikhegger**

### CODE REVIEW

File: cob_bringup/drivers/hokuyo.launch [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
 
-	<!-- load parameter -->
-	<rosparam ns="$(arg name)" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg name).yaml"/>
 
 	<!-- start laser driver -->
-	<node ns="$(arg name)" pkg="hokuyo_node" type="hokuyo_node" name="driver" respawn="false" output="screen">
-		<remap from="scan" to="scan_raw"/>
+	<node pkg="hokuyo_node" type="hokuyo_node" name="$(arg name)" respawn="false" output="screen">
+    	<!-- load parameter -->
+    	<rosparam command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg name).yaml"/>
```
Comment (fmessmer) [inline cob_bringup/drivers/hokuyo.launch:12]: indentation!

**Comment by fmessmer** [comment]
So with this PR, what will the names of the topics be?
Should be `base_laser_top/scan_raw`, right?

As for `cob4` the sick laser scanner topics are:

```
/base_laser_front/scan_raw
/base_laser_left/scan_raw
/base_laser_right/scan_raw
```

**Comment by fmessmer** [comment]
I think we already fixed this for the [simulated hokuyo](https://github.com/ipa320/cob_common/blob/indigo_dev/cob_description/urdf/sensors/hokuyo_lx30_laser.gazebo.xacro)
Could you verify and comment whether topic names are consistent for your robot both in simulation and hardware?

**Comment by fmessmer** [comment]
@frederikhegger 
Any update on this one?
Did you compare with simulation yet?

**Comment by frederikhegger** [comment]
I just compared the namespace with the one from the SICKs and they would be different. So do not merge.

**Pull request closed without merging**